src/ocean/text/convert/Formatter.d

```diff
@@ -86,7 +86,7 @@ public istring format (Args...) (cstring fmt, Args args)
 
 *******************************************************************************/
 
-public mstring sformat (Args...) (ref mstring buffer, cstring fmt, Args args)
+public mstring aformat (Args...) (ref mstring buffer, cstring fmt, Args args)
```

Comment:
We cannot use it because it's ambiguous. Considering the signatures:
```D
public istring format (Args...) (cstring fmt, Args args);
public mstring [s]format (Args...) (ref mstring buffer, cstring fmt, Args args);
```

The call is ambiguous in D1 in the following situation:
```D
istring fmt = "This is an {}";
format(fmt, "lvalue");
// ^ Any additional arguments are also ambiguous
```

It's also ambiguous with a mutable format string in D2.